### PR TITLE
ept_test: Fix CmdError

### PR DESCRIPTION
**Pull request by huangyum**
ID: 1878589
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by huangyum** [comment]
(1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.ept_test.q35: PASS (109.03 s)

**Comment by huangyum** [comment]
@vivianQizhu Would you please help review, thanks.

File: qemu/tests/ept_test.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def run(test, params, env):
     unload_cmd = params["unload_cmd"]
     load_cmd = params["load_cmd"]
     read_cmd = params["read_cmd"]
-    ept_value = process.system_output(read_cmd % "ept")
+    ept_value = process.system_output(read_cmd % "ept").decode()
```
Comment (vivianQizhu) [inline qemu/tests/ept_test.py:25]: Or `process.getoutput()`, which does not require extra `decode()`.

Comment (huangyum) [reply qemu/tests/ept_test.py]: Yes, you are right. Updated.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**